features/flows/elastic/src/test/java/org/opennms/netmgt/flows/elastic/DocumentEnricherTest.java

```diff
@@ -93,4 +96,30 @@ private static OnmsNode createOnmsNode(int nodeId, String foreignSource) {
         node.setForeignId(nodeId + "");
         return node;
     }
+
+    @Test
+    public void testCreateClassificationRequest() {
+        final FlowDocument flowDocument = new FlowDocument();
+
+        // verify that null values are handled correctly, see issue HZN-1329
+        ClassificationRequest classificationRequest;
+
+        classificationRequest = enricher.createClassificationRequest(flowDocument);
+        assertFalse(classificationRequest.isClassifiable());
```

Comment:
Can we either use `assertEquals` or `assertThat`. `assertTrue`  or `assertFalse` are not ideal when they fail, as the usual error messages not telling you what exactly is wrong, whereas the others at least tell you something like `actual true, expected false`.